driver/conv_driver.hpp

```diff
@@ -1019,9 +1019,12 @@ int ConvDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
                 break;
```

Comment:
The way you use layout utilities looks good. I'm not a native MIOpen developer so abstain from the rest of code changes. I have two feedbacks:
- Could you add a test stage that use ctest to test specifically NHWC layout, in particular with 1x1 and 3x3 kernels. Some configs to choose from [here](http://confluence.amd.com/pages/viewpage.action?spaceKey=MLSE&title=Convolution+Models+Performance+Optimization#ConvolutionModelsPerformanceOptimization-Resnet50_v1.5,TensorflowNVexample,FP16BS256) are probably good candidates.
- Could you make sure `MIOPEN_ENABLE_LOGGING_CMD=1` populates the correct logging command? This is such that when framework people take your PR and test, they have a good baseline to compare with.

Reply:
Hi @jerryyin ,
1. I added a test to test gpu_reference_kernel specifically, with NHWC layout, [in this file](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/728/files#diff-b8f24fcbf6af64fb36fd0cacded514848f06dc3046c6008edd9f293b706ad1a5). This is a ctest and will run while `make check`. But I think you are right, I need at least add a flag to parse in NHWC layout info for test_conv2d. Maybe follow the [MIOpenDriver command line flags]( https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/driver/conv_driver.hpp#L570)
2. `MIOPEN_ENABLE_LOGGING_CMD=1` sure I'll add layout info of this senario.

Reply:
Hi @jerryyin  @atamazov for add layout print when `MIOPEN_ENABLE_LOGGING_CMD=1`, I updated the changes and now it support it. For test_conv2d to support NHWC layout, it seems need some refactor of test driver structure (it heavily utilize template and may need more time to evaluate), I think maybe left that change in another PR

Reply:
> For test_conv2d to support NHWC layout, it seems need some refactor of test driver structure (it heavily utilize template and may need more time to evaluate ), I think maybe left that change in another PR.

Sounds good to me. Once you wrapped up this PR, could you prioritize on working on that? It makes generic NHWC testing possible with ctest utility. Thanks!

Reply:
@jerryyin OK

Reply:
Me && Carlus talked offline. Carlus will work on adding NHWC support to ctest starting 2/17. Marking this as resolved.